README.md

```diff
@@ -1 +1,33 @@
-# gym-project
\ No newline at end of file
+# Capstone-project-gym
+
+> This is the first Milestone project that I am going to build which is A website for a gym. This project is created with HTML, CSS AND JS and has some functionality like the hamburger menu, adapting to different screen sizes(Mobile and Desktop Version). For now only the [Home] and [about] page is working. I special thanks to Mr. Cindy Shin for lating as to use their website design.
+
+## Built With
```

Comment:
**[Optional]**
- [ ] Please add an 5 mins introductory loom video of your project. This is your documentation file so it should contain an intro video of your application. :+1: